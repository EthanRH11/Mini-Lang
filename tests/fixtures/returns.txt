
proc calculateArea(int width, int height) => {
    int area = width * height;

    result => {area};
}

proc myFunction(int x) => {

    result => {x + 3};
}

proc ifFunction(int x) => {
    if(x < 0){
        result => {-1};
    } else {
        result => {x * 2};
    }
}

begin:
    out_to_console(calculateArea(5, 3)); ...
    out_to_console(myFunction(7)); ...
    out_to_console(ifFunction(-2)); ...
    out_to_console(ifFunction(4)); 

end