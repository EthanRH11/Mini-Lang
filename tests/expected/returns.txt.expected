15
10
-1
8